Neos.Neos/Tests/Behavior/Features/ContentRepository/Security/WorkspacePermissions.feature

```diff
@@ -112,15 +100,62 @@ Feature: Workspace permission related features
       | collaborator |
       | owner        |
 
+  Scenario: Creating a workspace without Neos User but READ permissions on live
+    Given I am not authenticated
+    And the shared workspace "some-shared-workspace" is created with the target workspace "live"
+
+  Scenario Outline: Creating a workspace with READ permissions (on live)
+    Given I am authenticated as <user>
+    And the shared workspace "some-shared-workspace" is created with the target workspace "live"
+
+    And the personal workspace "some-other-personal-workspace" is created with the target workspace "live" for user <user>
+
+    Examples:
+      | user              |
+      | admin             |
+      | owner             |
+      | collaborator      |
+      | uninvolved_editor |
+      | restricted_editor |
+      | simple_user       |
```

Comment:
basically theses test means that EVERYBODY can create a new workspace based on live, not even a neos user is required - im not sure if this is correct, but in 8.3 i also could find no restriction for `new Workspace` or adding that to the repository.